package.json

```diff
@@ -38,7 +38,9 @@
     "chai": "^3.5.0",
     "mocha": "^3.0.2",
     "webpack": "^1.13.0",
-    "uglify-js": "^3.0.10"
+    "uglify-js": "^3.0.10",
+    "wnpm-ci": "*",
```

Comment:
Using `npx` no need for installing

```suggestion
```

Reply:
Generally you'd still specify version ranges here, and npx would run them from `node_modules` if they didn't match the global version.  Since this already wasn't defined though, it could make sense to leave that issue separate from this PR

This change suggestion has the wrong two lines though - will do it as a separate edit

Reply:
Sorry my bad, but you understood me 🙈 

No need for installation if we want to use latest with npx 🏂